content/blog/2019/istio1.1_perf/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Cloud service management comes of age with Istio v1.1 performance update
+description: An overview of Istio 1.1 performance.
+publishdate: 2019-03-18
+subtitle: An overview of Istio 1.1 performance improvements
+attribution: Surya V Duggirala (IBM), Mandar Jog (Google), Jose Nativio (IBM)
+keywords: [Kubernetes, Istio, Docker, Container, performance, service mesh]
```

Comment:
Please remove all spaces and make everyrhing lowercase.

Look at other keywords in the repo to make sure you're using consistent keywords to get better See Also links throughout the site.